scripts/assess-strat/parse_results.py

```
@@ -0,0 +1,200 @@
+            failed_parse.append(key)
+            continue
+
+        title = extract_title(text)
+        rows.append({
+            "ID": key,
+            "Title": title,
+            **scores,
+        })
+
+    # Write CSV
+    fieldnames = ["ID", "Title"] + CRITERIA + ["Total", "Verdict", "Needs_Attention"]
```

Ruff (0.16.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)